internal/pkg/template/templates/environment/cf.yml

```diff
@@ -523,6 +523,16 @@ Outputs:
     Value: !GetAtt InternalLoadBalancer.DNSName
     Export:
       Name: !Sub ${AWS::StackName}-InternalLoadBalancerDNS
+  InternalLoadBalancerFullName:
```

Comment:
Are these exports planning to be used somewhere? 🤔

Reply:
half of them are 😄 `InternalLoadBalancerHostedZone` is already used [here](https://github.com/aws/copilot-cli/pull/3622/files#diff-ebcd538b6b6f76a2a339b6bcf0d0dbc47b1e1be7bbbcf7ba01c8338e48fefc4dR37) and `InternalLoadBalancerFullName` will be needed if we support autoscaling by requests (see [here](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/template/templates/workloads/services/lb-web/cf.yml#L100)). I just added it today so that people won't have to upgrade their env when (if?) we add that feature, but I'm fine to remove that one.

Reply:
nvm lol